wandb/sdk/launch/launch_add.py

```diff
@@ -174,6 +174,7 @@ def _launch_add(
             launch_spec["job"] = None
 
         launch_project = create_project_from_spec(launch_spec, api)
+        config = config or {}
         docker_image_uri = build_image_from_project(launch_project, api, config)
```

Comment:
maybe just set a default in the function def?

or:

```suggestion
        docker_image_uri = build_image_from_project(launch_project, api, config or {})
```